### PR TITLE
add description for new webUI

### PR DESCRIPTION
**Pull request by alexpeterson91**
i cant find a still or mock build page with it working properly as we spoke today it looks like it was tkaen down after the test, so i just wrote something simple for what i could build today with the latest dappmanager and this branch

### CODE REVIEW

File: dappnode_package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
...
-  "description": "Validate with prysm: a Go implementation of the Ethereum 2.0 Serenity protocol and open source project created by Prysmatic Labs. Beacon node which powers the beacon chain at the core of Ethereum 2.0\n\nIt includes a Grafana dashboard for the [DMS](http://my.dappnode/#/installer/dms.dnp.dappnode.eth) thanks to the amazing work of [metanull-operator](https://github.com/metanull-operator/eth2-grafana)",
+  "description": "Validate with prysm: a Go implementation of the Ethereum 2.0 Serenity protocol and open source project created by Prysmatic Labs. Beacon node which powers the beacon chain at the core of Ethereum 2.0\n\nIt includes a Grafana dashboard for the [DMS](http://my.dappnode/#/installer/dms.dnp.dappnode.eth) thanks to the amazing work of [metanull-operator](https://github.com/metanull-operator/eth2-grafana). This version of Prysm introduces a new method of authentication to access the Prysm WebUI.  The first time the package is started you will need to click on the link below next to the word token.  After opening the webUI with this token link you can access the webUI here at (prysm-prater.dappnode)[http://prysm-prater.dappnode] ",
```
Comment (pablomendezroyo) [inline dappnode_package.json:8]: I would add the tip to use the URL link with the JWT (on the first login) in the `getting_started.md`, so it appears as higher as possible
image

Comment (dapplion) [reply dappnode_package.json:8]: @pablomendezroyo Instructions have to be extremely clear on what to do. Consider using bold font or emojis to draw attention in a polite way

**Comment by alexpeterson91** [comment]
I’m gonna make a forum post and likely a short simple video. I’ll get to work after the call today. I literally just got  the latest version that’ll likely be published after adding the description built on my test server so I can see what it actually looks like to be able to be super clear and use screenshots and a video if needed

File: dappnode_package.json (hunk 1 above)
Comment (pablomendezroyo) [reply dappnode_package.json:8]: check also https://github.com/dappnode/DAppNodePackage-prysm-prater/pull/22#discussion_r754978486

Comment (pablomendezroyo) [reply dappnode_package.json:8]: Check https://github.com/dappnode/DAppNodePackage-prysm-prater/pull/22#issuecomment-976487501

**Comment by pablomendezroyo** [comment]
closed by https://github.com/dappnode/DAppNodePackage-prysm-prater/pull/22

**Pull request closed without merging**